dino.icu.yaml

```
@@ -100,6 +100,11 @@ emojile: # wordle with slack emojis - https://github.com/yodalightsabr/emojile
   ttl: 1
   type: CNAME
   value: yodalightsabr.github.io.
+  
+gaurav: # gaurav website
+  ttl: 1
+  type: CNAME
+  value: yednapg.vercel.app.
```

```suggestion
  value: cname.vercel-dns.com.
```

You'll also need to add the Vercel verification TXT record to `_vercel`